tests/regrtest_data/long_test_file.py

```diff
@@ -0,0 +1,100 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+print(1)
```

Comment:
Seems like this file is unrelated, right ?

Reply:
No it is used for the test. I needed a module that isn’t restricted by line numbers as the enable tests are using various line numbers to test the behaviour.
Using an empty module creates issue as you can’t disable something on line 10 if it doesn’t exist. Thus, I created an extra long file so we never run into issue with that.